forge-core/src/main/java/forge/card/CardChangedType.java

```diff
@@ -26,31 +28,30 @@
  * @version $Id: CardType.java 24393 2014-01-21 06:27:36Z Max mtg $
  */
 public class CardChangedType {
+
+    public enum Remove {
```

Comment:
If this is public, it probably should be a different file.

Reply:
:+1: Agreed